_api-reference/ingest-apis/processors/csv.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+title: CSV
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 40
+---
+
+# CSV
+
+The `csv` processor is used to parse comma-separated values (CSV) and store them as individual fields in a document. The processor ignores empty fields. The following is the syntax for the `csv` processor: 
+
+```json
+{
+  "csv": {
+    "field": "field_name",
+    "target_fields": ["field1, field2, ..."]
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `csv` processor.
+
+**Parameter** | **Required** | **Description** |
+|-----------|-----------|-----------|
+`field`  | Required  | Name of the field where the data should be converted. Supports template snippets.|
+`target_fields`  | Required  | Name of the field to store the parsed data in. |
+`value`  | Required  | Value to be appended. This can be a static value, a dynamic value derived from existing fields, or a value obtained from external lookups. Supports template snippets. | 
+`description`  | Optional  | Brief description of the processor.  |
+`empty_value`  | Optional  | Represents optional parameters that are not required to be present or are not applicable.  |
```

Comment:
The description is not correct. This is a default value to put when value does not exist.